pywatts/modules/preprocessing/custom_scaler.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict
+
+import xarray as xr
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions import WrongParameterException
+
+
+class CustomScaler(BaseTransformer):
+    """
+    Scaling step to scale a time series individually by a multiplier and a bias.
+    By default the scaling does not affect the time series, i.e., the multiplier is 1.0 and the bias 0.0.
+    """
+
+    def __init__(self, multiplier: float = None, bias: float = None, name: str = "CustomScaler"):
+        """ Initialize the scaling step.
+        :param multiplier: Value that is multiplied to every value in the time series.
+        :type multiplier: float, optional
+        :param bias: Value that is added to every value in the time series
+        :type bias: float, optional
+        """
+        super().__init__(name)
+        self.has_inverse_transform = True
+
+        if multiplier == 0:
+            raise WrongParameterException(
+                "Multiplication by 0 is not possible.",
+                "During initialisation set a multiplier different to 0.",
+                module=self.__class__)
+
+        if multiplier:
+            if multiplier == 0:
+                raise WrongParameterException(
+                    "Multiplication by 0 is not possible.",
+                    "During initialisation set a multiplier different to 0.",
+                    module=self.__class__)
+            self.multiplier = multiplier
+        else:
+            self.multiplier = 1.0
```

Comment:
It seems to me that you want to set a default value. You should set a default in the definition of the function. I.e., replace multiplier:float =None in line15 by mulitplier:float = 1.0

Then you can also delete line 31 to 39.